backend/spec/controllers/spree/admin/reimbursements_controller_spec.rb

```diff
@@ -38,7 +38,10 @@
     let(:order) { customer_return.order }
     let(:return_item) { customer_return.return_items.first }
     let(:payment) { order.payments.first }
-    before { return_item.receive! }
+    before do
+      return_item.skip_customer_return_processing = true
```

Comment:
Do you think it could be a good idea to add this as default in the return item factory?

Reply:
I totally agree 👍 
This also reduced the changes I had to make to spec files